lib/matplotlib/axes/_base.py

```diff
@@ -680,7 +680,7 @@ def __init__(self, fig,
             raise ValueError('Width and height specified must be non-negative')
         self._originalPosition = self._position.frozen()
         self.axes = self
-        self._aspect = 'auto'
+        self._aspect = None
```

Comment:
```suggestion
        self._aspect = mpl.rcParams['axes.aspect']
```
Resolution should happen at initialization.

Reply:
Fixed.